These two tests failed
```
======================================================================
ERROR: test_motor_on_for_degrees (__main__.TestAPI)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/api_tests.py", line 182, in test_motor_on_for_degrees
    m.on_for_degrees(None, 100)
  File "tests/../ev3dev2/motor.py", line 909 in on_for_degrees
    speed_sp = self._speed_native_units(speed)
  File "tests/../ev3dev2/motor.py", line 864, in _speed_native_units
    assert -100 <= speed <= 100,\
TypeError: unorderable types: int() <= NoneType()
======================================================================
ERROR: test_motor_on_for_rotations (__main__.TestAPI)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/api_tests.py", line 201, in test_motor_on_for_rotations
    m.on_for_rotations(None, 5)
  File "tests/../ev3dev2/motor.py", line 893, in on_for_rotations
    speed_sp = self._speed_native_units(speed)
  File "tests/../ev3dev2/motor.py", line 864, in _speed_native_units
    assert -100 <= speed <= 100,\
TypeError: unorderable types: int() <= NoneType()
```
Do we officially support accepting None for speed, rotations, degrees, or seconds?  I did not test these when I was running through all the permutations.

--- a/tests/api_tests.py
+++ b/tests/api_tests.py
@@ -177,14 +177,6 @@ class TestAPI(unittest.TestCase):
         self.assertEqual(m.speed_sp, 0)
         self.assertEqual(m.position_sp, 0)
 
-        # None speed
-        with self.assertRaises(ValueError):
-            m.on_for_degrees(None, 100)
-        
-        # None distance
-        with self.assertRaises(ValueError):
-            m.on_for_degrees(75, None)
-    
     def test_motor_on_for_rotations(self):
         clean_arena()
         populate_arena([('large_motor', 0, 'outA')])
@@ -195,14 +187,6 @@ class TestAPI(unittest.TestCase):
         m.on_for_rotations(75, 5)
         self.assertEqual(m.speed_sp, int(round(0.75 * 1050)))
         self.assertEqual(m.position_sp, 5 * 360)
-
-        # None speed
-        with self.assertRaises(ValueError):
-            m.on_for_rotations(None, 5)
-        
-        # None distance
-        with self.assertRaises(ValueError):
-            m.on_for_rotations(75, None)
 
     def test_move_tank_relative_distance(self):
         clean_arena()

